src/index.js

```diff
@@ -85,6 +86,9 @@ window.wikiaJWPlayer = function (elementId, options, callback) {
 					autoplay: options.autoplay,
 					selectedCaptionsLanguage: options.selectedCaptionsLanguage,
 					i18n: i18n
+				},
+				wikiaWatermark: {
+					show: true//!!options.videoDetails.playlist[0].watermark
```

Comment:
I think this option should just activate watermark plugin. If plugin is active then show watermark if `watermark` custom parameter is set. If plugin is not active we shouldn't show watermark even if the custom parameter is set.